control-plane/README.md

```diff
@@ -16,6 +16,19 @@ The job control plane is used for controlling and monitoring aggregation jobs.
 * Run `firebase use --add` to add an alias. When prompted, select your **Project ID**, then give your Firebase project an alias. For more info, visit this [link](https://firebase.google.com/docs/cli#add_alias).
 * Run `yarn start` or `yarn deploy`
 
+## Making an Admin Account
```

Comment:
We should explain the roles and how the auth/authz works, i.e. users can only browse the jobs after the admin grants permissions.